README.md

```diff
@@ -51,6 +51,7 @@ painless as possible when issues arise.
 - [Warehouse.ai Internals](#warehouseai-internals)
   - [Data Models](#data-models)
   - [Config options](#config-options)
+- [Getting a package `warehouse.ai`-ready](#https://github.com/godaddy/warehouse.ai/tree/package-docs#getting-a-package-warehouseai---ready)
```

Comment:
Should this be here in the readme or as a separate doc, will there be enough related bits to justify a separate doc? (setting up auth for scopes, `.npmrc` setup, what you have, other stuff?)